### ISSUE TITLE
option profiles

### ISSUE THREAD

**Issue by paponius** [gildas-lormeau/SingleFile#276]
**Describe the bug**
Settings are not saved per Option Profile (initially - on fresh install)

**To Reproduce**
Switching profile on Option page then changing some options,
then again switching to another profile or back to Default.
Either are Options saved under all profiles. Or Options are unaffected by profile selection.

But if Reset button on bottom is pressed and a profile (any) is reset, it all starts to work as expected.

**Expected behavior**
Changing Option Profile will change setting on Option page respectfully to changed profile

**Screenshots**

**Environment**
 - OS: Win10 Pro
 - Browser: Firefox
 - Version: 64

**Comment by gildas-lormeau** [comment]
Thank you for having reported this issue. I was able to reproduce it and fix it. The fix is available in the version 1.11.45 I've just published;

**Comment by paponius** [comment]
Thanks for the fix and the plugin.
I can confirm it's working as expected in a fresh firefox.
Also in my firefox for daily use after the plugin update.
There is one exception, but it might affect just a few users,
if new Setting Profile was created before, in a version of the plugin from before the update,
then this one Setting Profile is still affected by the issue.
Creating another Setting Profile after update will make the later work ok, but not the one created before.

**Comment by gildas-lormeau** [comment]
Thanks for the info @paponius. Fortunately, this issue lasted only some days. Exporting and importing the settings should help to fix the issue for the affected users.

**Comment by paponius** [comment]
The Setting Profile, which I created before the update is still "linked" to Default Profile. Meaning a setting changed in either of them is reflecting into the other.
Newly created Setting Profile, after the update, behaves as expected. A changed setting in it is only changed in that Setting Profile. 
I know deleting this affected profile would fix the issue. I am just reporting my observations. A user, who wanted to use Setting Profile and created it before while it was not working, might not know the issue was resolved and steps required for it to start working as was expected.
But then again, it's a minor problem, affecting possibly just a few users, and might be a waste of time adding a fix for such case.
